### PR TITLE
add ps tt commons to normalize

### PR DESCRIPTION
**Pull request by jaketrent**
- [x] Need to go back and fix TODOs (with real Cloudfront URL of font)
- Decided not to major version bump core or normalize. Reasoned that we could push the new packages to people transparently and remain backwards compatible with Gotham. This will help us transition faster. Gotham and the adjusted PS TT Commons are meant to be as identical in size and proportions as possible as to not require much visual tweaking. Tweaks that might need to be made are anticipated to be small and manageable.

### CODE REVIEW

File: packages/core/css/type.module.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 :root {
-  --psTypeFontFamily: "Gotham SSm A", "Gotham SSm B", sans-serif;
-  --psTypeFontFamilyCode: "Source Code Pro", monospace;
+  /* prettier-ignore */
+  --psTypeFontFamily: 'PS TT Commons', 'Gotham SSm A', 'Gotham SSm B', sans-serif;
```
Comment (danethurber) [inline packages/core/css/type.module.css:3]: 👍 nice. didn't think about falling back to gotham but good idea.

@sweeting i'm wondering what the ie11 fallback should be

Comment (jaketrent) [reply packages/core/css/type.module.css:3]: Thought we had said that IE gets sans-serif.

Comment (danethurber) [reply packages/core/css/type.module.css]: we did. i was just wondering if there was something more specific. like 'Arial'...

Comment (jaketrent) [reply packages/core/css/type.module.css]: Gotcha. I like going straight to sans-serif just to get the system font that's OS-specific.  

That's what `sans-serif` is to me conceptually, but I did also see that there's a new `system-ui` option, which I don't yet understand how it differs.

Comment (sweeting) [reply packages/core/css/type.module.css]: Arial might be a less noticeable difference as fallback for IE 11 than the Windows System font Segoe UI (a humanist sans).

Sample Page - PS TT Commons

Sample Page - Segoe UI

Sample Page - Arial

**Comment by jaketrent** [comment]
Still ready for review.  Red tests are unrelated.  Polyfill fix will come after merge of https://github.com/pluralsight/design-system/pull/1485

**Review by danethurber — APPROVED** [review]

**Pull request merged**